### PR TITLE
Adding specs in Web Standards

### PR DESCRIPTION
**Pull request by afonsopacifer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,13 @@ A curated list of awesome Web Components tools, articles and resources. Inspired
 
 ## Web Standards
 
+*Specs*
```
Comment (obetomuniz) [inline README.md]: @afonsopacifer Can you improve this syntax? The list are using header titles with 3 level for subsections, then use:

## Web Standards

 *Resources about the web standards under Web Components*

* [Item 1]()
* [Item 2]()
* [Item 3]()
* ...

### W3C Specifications

* [Item 1]()
* [Item 2]()
* [Item 3]()
* ...

**Comment by afonsopacifer** [comment]
I dont write "W3C" because the link to the specification of templates goes to the WHATWG

**Comment by obetomuniz** [comment]
Dont worry... Was an example :)

**Comment by afonsopacifer** [comment]
To keep the standard , 
I added a subtitled the subsection,
and synchronized with the original repository .

**Pull request merged**